From a53a20683da3b3bda0647a9859de66a839b4bebf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Ol=C5=A1=C3=A1k?= <marek.olsak@amd.com>
Date: Tue, 31 Jan 2017 15:13:35 +0530
Subject: [PATCH 2/4] radeonsi: silence runtime warnings with LLVM 3.9

Such as:
Warning: LLVM emitted unknown config register: 0x4

This is a non-intrusive back port of commit 0f7a6ea5e7b.
---
 src/gallium/drivers/radeonsi/si_shader.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/gallium/drivers/radeonsi/si_shader.c b/src/gallium/drivers/radeonsi/si_shader.c
index aad7dd0..9cfecb9 100644
--- a/src/gallium/drivers/radeonsi/si_shader.c
+++ b/src/gallium/drivers/radeonsi/si_shader.c
@@ -5862,6 +5862,9 @@ void si_shader_binary_read_config(struct radeon_shader_binary *binary,
 			conf->scratch_bytes_per_wave =
 				G_00B860_WAVESIZE(value) * 256 * 4 * 1;
 			break;
+		case 0x4:
+		case 0x8:
+			break; /* just spilling stats, not important */
 		default:
 			{
 				static bool printed;
-- 
2.7.4

